### PR TITLE
Update helloworld os

### PR DESCRIPTION
**Pull request by hgarvison**
Update helloworld-aci OS to alpine 3.19

### CODE REVIEW

File: hello-world/ACI/app/main.py [hunk 1]
```diff
@@ -77,7 +72,7 @@ def is_hex(x): return all(c in string.hexdigits for c in x)
     # put everything together
     return (
         style +
-        "<div>" + "<h1>Welcome to Confidential containers on Azure Container Instances!</h1>" +
+        "<div>" + "<h1>Welcome to Confidential ontainers on Azure Container Instances!</h1>" +
```
Comment (SethHollandsworth) [inline hello-world/ACI/app/main.py:75]: I think this should get reverted?

Comment (hgarvison) [reply hello-world/ACI/app/main.py:75]: woops I thought I typed the C back, good catch

File: hello-world/ACI/arm-template.json [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+      ],
+      "metadata": {
+        "description": "The behavior of Azure runtime if container has stopped."
+      }
+    }
+  },
+  "resources": [
+    {
+      "type": "Microsoft.ContainerInstance/containerGroups",
+      "apiVersion": "2022-10-01-preview",
```
Comment (ksayid) [inline hello-world/ACI/arm-template.json:63]: nit: should the `apiVersion` be updated to a non-preview version?

Comment (hgarvison) [reply hello-world/ACI/arm-template.json:63]: Yeah I just used the original version of the template for hello world, but I can update it

**Review by ksayid — APPROVED** [review]
LGTM

**Pull request merged**